torch/csrc/jit/ir.h

```diff
@@ -968,6 +968,10 @@ struct Graph {
 
   Graph() : Graph(c10::make_intrusive<Scope>()) {}
 
+  const std::unordered_set<const Value*>& values() {
```

Comment:
Code is dead as far as I can tell. This must be deleted. If left in, it will lead to a bunch of people misusing it in incredibly awful ways. They will forget that Blocks may exist. They will forget the inputs to the graph will exist. They will forget the order of this thing is undetermined and create a bunch of non-deterministic code.